flink-python/pyflink/table/table_environment.py

```diff
@@ -98,18 +99,25 @@ def __init__(self, j_tenv, serializer=PickleSerializer()):
         self._open()
 
     @staticmethod
-    def create(environment_settings: EnvironmentSettings) -> 'TableEnvironment':
+    def create(conf_or_settings: Union[EnvironmentSettings, Configuration]) -> 'TableEnvironment':
         """
         Creates a table environment that is the entry point and central context for creating Table
         and SQL API programs.
 
-        :param environment_settings: The environment settings used to instantiate the
+        :param conf_or_settings: The configuration or environment settings used to instantiate the
                                      :class:`~pyflink.table.TableEnvironment`.
         :return: The :class:`~pyflink.table.TableEnvironment`.
         """
         gateway = get_gateway()
-        j_tenv = gateway.jvm.TableEnvironment.create(
-            environment_settings._j_environment_settings)
+        if isinstance(conf_or_settings, EnvironmentSettings):
+            environment_settings = conf_or_settings
+        elif isinstance(conf_or_settings, Configuration):
+            environment_settings = EnvironmentSettings.new_instance() \
```

Comment:
EnvironmentSettings.from_configuration(conf_or_settings)

Reply:
`from_configuration` is deprecated